services.py

```diff
@@ -1,51 +1,57 @@
-import os
+from datetime import date
+import json
 import requests
 import subprocess
 import configparser
 import geocoder
 import ast
 import logging
-from geopy.geocoders import Nominatim
+from geopy.geocoders import Photon
+import serial
+
+SERIAL_PORT = "/dev/ttyUSB0"
+
+
+logging.basicConfig(filename=f'./assets/logs/{date.today()}.log', level=logging.DEBUG)
+
+logger = logging.getLogger('services')
+logger.setLevel(logging.DEBUG)
 
 class Services:
     def __init__(self):
-        self.logger = self.get_logger('services')
+
         self.formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
 
     def run_bash(self, script_path):
         try:
             subprocess.run(["bash", script_path], capture_output=True)
-            self.logger.info('Script executed successfully')
+            logger.info('Script executed successfully')
         except subprocess.CalledProcessError as e:
-            self.logger.error(f'Error running bash script: {e}')
+            logger.error(f'Error running bash script: {e}')
 
     def run_python(self, script_path):
         try:
             with open(script_path, 'r') as f:
                 script = f.read()
             exec(script)
-            self.logger.info('Script executed successfully')
+            logger.info('Script executed successfully')
         except Exception as e:
-            self.logger.error(f'Error running python script: {e}')
+            logger.error(f'Error running python script: {e}')
```

Comment:
If the script is determined by shebang, you don't need 2 different functions.

Reply:
👍Was going to do this and forgot.